src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -77,15 +76,15 @@ const SubsidiesSummary = ({
 
   return (
     <>
-      {hasAssignedSubsidyOrRequests && (
+      {hasAssignedSubscriptionOrCodeOrOffer && (
```

Comment:
nit: is this conditional necessary if the component already returns `null` above when `hasAssignedSubscriptionOrCodeOrOffer` is falsey?